### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by vick3d**
Scaffolds new React Native application

Task:
Initialize Expo

Add Node_modules and expo to gitignore

### CODE REVIEW

**Review by yanqiuzhang — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+.expo/
```
Comment (magnus-thor) [inline .gitignore]: add `.idea` to gitignore

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (magnus-thor) [inline .idea/codeStyles/Project.xml]: remove the .idea folder from git

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (vick3d) [reply .gitignore]: fixed

File: .idea/codeStyles/Project.xml (hunk 2 above)
Comment (vick3d) [reply .idea/codeStyles/Project.xml]: fixed

**Review by magnus-thor — APPROVED** [review]

**Pull request merged**